oryx-app/src/main/java/com/cloudera/oryx/app/speed/kmeans/KMeansSpeedModelManager.java

```diff
@@ -51,7 +64,11 @@ public void consume(Iterator<KeyMessage<String, String>> updateIterator) throws
       String message = km.getMessage();
       switch (key) {
         case "UP":
-          // do nothing;
+          List<?> update = MAPPER.readValue(message, List.class);
```

Comment:
This layer won't get updated from the batch layer. It can just ignore them. In fact it will just be hearing its own updates.